rustls/src/lib.rs

```diff
@@ -1,41 +1,66 @@
 //! # Rustls - a modern TLS library - portable-rustls fork
 //!
```

Comment:
I think removing this line which now comes before a pair of TODO comment lines should help reduce the diff here
```suggestion
```